src/main/resources/com/toptrumps/online/views/_parts/footer.ftl

```diff
@@ -1,2 +1,8 @@
+	<!-- Import JQuery, as it provides functions you will probably find useful (see https://jquery.com/) -->
+    <script src="/assets/scripts/jquery-3.4.1.min.js"></script>
+	<script src="/assets/scripts/game.js"></script>
+	<script>
+		const restAPIurl = 'http://localhost:7777/toptrumps';
```

Comment:
We should be careful with "localhost" -- sometimes it's better to just write the local IP.